### DISCUSSION TITLE
[Question]: HAOS Addin for Frigate vs HACS Custom Integration - feature differences?

### DISCUSSION THREAD

**Discussion by wedwardbeck** [blakeblackshear/frigate#12189] [Ask A Question] [answered]
### What is your question:

I'd like to know the key feature differences between running the HAOS Addin and using the HACS Integration for Frigate.  I'm running HAOS and have Frigate integrated via HACS now.  I just set it up and am curios if there are any key features I may be missing instead of using the add-in.  I'm good using the standalone docker instance, but I'm also looking for the least admin overhead since this is one of many side projects.  TIA

**Comment by NickM-27** [comment, marked as answer]
the addon and the integration are entirely different things, like it says in the docs. 

![Screen Shot 2024-06-28 at 12 27 14 PM](https://github.com/blakeblackshear/frigate/assets/14866235/6862957c-654f-4a4b-803c-5d5c39794924)

if you are comparing running externally in docker vs running in an addon then you only lose features going with the addon

**Reply by wedwardbeck** [reply]
thanks - "then you only lose features going with the addon" was the info I was looking for.
